testing/logging/test_reporting.py

```diff
@@ -827,6 +828,40 @@ def test_log_file(request):
         assert "This log message won't be shown" not in contents
 
 
+def test_log_file_unicode(testdir):
+    log_file = testdir.tmpdir.join("pytest.log").strpath
+
+    testdir.makeini(
+        """
+        [pytest]
+        log_file={}
+        log_file_level = INFO
+        """.format(
+            log_file
+        )
+    )
+    testdir.makepyfile(
+        """
+        import logging
+        def test_log_file():
+            logging.getLogger('catchlog').info("Normal message")
+            logging.getLogger('catchlog').info("\u251c")
+            logging.getLogger('catchlog').info("Another normal message")
+    """
+    )
+
+    result = testdir.runpytest()
+
+    # make sure that that we get a '0' exit code for the testsuite
+    assert result.ret == 0
+    assert os.path.isfile(log_file)
+    with open(log_file, encoding="utf-8") as rfh:
+        contents = rfh.read()
+        assert "Normal message" in contents
+        assert "\u251c" in contents
```

Comment:
this is a little misleading, in python2 this will be a literal `"\u251c"` -- in python3 it'll be the character `├`

I suspect this test didn't actually fail on python2 before your patch -- mind changing the test to write unicode in both python2 and python3?

Reply:
Thanks, you were right to the point. I updated the tests to solve this issue.